File: AkkaWithZeroMQ/src/main/java/org/akka/essentials/zeromq/example3/server/MyServerActorSystem.java
package org.akka.essentials.zeromq.example3.server;

import akka.actor.ActorSystem;
import akka.actor.Props;

public class MyServerActorSystem {

	/**
	 * @param args
	 */
	public static void main(String[] args) {
		ActorSystem system = ActorSystem.create("zeromqServerTest");
		system.actorOf(new Props(ServerActor.class), "server");
	}

}

File: AkkaWithZeroMQ/src/main/scala/org/akka/essentials/zeromq/example3/server/MyServerActorSystem.scala
package org.akka.essentials.zeromq.example3.server
import akka.actor.ActorSystem
import akka.actor.Props

object MyServerActorSystem {

  def main(args: Array[String]): Unit = {
    val system = ActorSystem("zeromqServerTest")
    system.actorOf(Props[ServerActor], name = "server")
  }
}